netbox_cmdb/netbox_cmdb/models/route_policy.py

```diff
@@ -27,7 +27,7 @@ class RoutePolicy(ChangeLoggedModel):
     objects = RestrictedQuerySet.as_manager()
 
     def __str__(self):
-        return str(self.name)
+        return f"{self.device}-{self.name}"
```

Comment:
is it expected that you change that part ? we might use it on route-map gui views so appending the device name might result in something dirty

Reply:
yes, it is part of the fifth commit in this PR: refactor(model): better string representation 

We did that for the other models.

Reply:
> we might use it on route-map gui views

it does not seem to cause any issues with the prefix-lists and community-lists, which do have the same `__repr__` and `__str__`.